Modules/Operations/Routes/web.php

```diff
@@ -14,4 +14,7 @@
 Route::prefix('operations')->group(function () {
     Route::get('/office-location', 'OperationsController@index')->name('office-location.index');
     Route::post('/office-location', 'OperationsController@store')->name('office-location.store');
+    Route::get('/office-location/{id}/edit', 'OperationsController@edit')->name('office-location.edit');
+    Route::put('/office-location/{id}/update', 'OperationsController@update')->name('office-location.update');
+    Route::delete('/office-location/{id}', 'OperationsController@destroy')->name('office-location.destroy');
```

Comment:
Please update the all places.
```suggestion
    Route::delete('/office-location/{id}', 'OperationsController@delete')->name('office-location.delete');
```

Reply:
I have done.